Modules/@babylonjs/react-native-iosandroid/android/build.gradle

```diff
@@ -41,10 +41,6 @@ if (project.hasProperty("BASEKIT_BUILD")) {
     basekitBuild = project.getProperty("BASEKIT_BUILD")
 }
 
-if (REACT_VERSION >= 71) {
-    DEFAULT_NDK_VERSION = '23.1.7779620'
```

Comment:
Will this still work with older RN versions? Or will there be some ndk incompatibility?